### DISCUSSION TITLE
Add LEDs for FLAGF, FLAGO, and RTN

### DISCUSSION THREAD

**Discussion by wrh3c** [nicolacimmino/PLC-14500#3] [Ideas]
If you ever redesign the board, I suggest adding LEDs for FLAGF, FLAGO, and RTN (MC14500 pins 9, 10, and 11).  These LEDs can be seen when executing NOPF, NOPO, and RTN instructions.

There were three extra LEDs with my kit, so I connected them (with current-limiting resistors) from these pins to GND and they work as expected!

**Comment by nicolacimmino** [comment]
I had left them out originally as they are not wired to perform any function. On the other hand, considering it's a product to familiarize with the MC14500 it might make sense to include. At the moment I have quite a stock of boards but, yeah, if I order more I'll consider
